docs/features/software-templates/writing-custom-field-extensions.md

```diff
@@ -82,7 +82,7 @@ import {
   createScaffolderFieldExtension,
 } from '@backstage/plugin-scaffolder';
 import { MyCustomExtension } from './MyCustomExtension';
```

Comment:
```suggestion
import { MyCustomExtension, myCustomValidation } from './MyCustomExtension';
```

Reply:
You can remove the duplicate line this way :)

Reply:
resolved as expected